### PR TITLE
Visual design refinements

### PR DESCRIPTION
**Pull request by creuter**
* Use CSS Grid for layout
* Add homepage illustration

**Small:**

localhost-4000- iphone 6

**Medium:**

localhost-4000- ipad

**Large:**

localhost-4000- laptop with mdpi screen

### CODE REVIEW

**Comment by creuter** [comment]
Most of the markup changes are removing wrapping `<section>`s

File: web/templates/home_page/about.html.eex [hunk 1]
```diff
@@ -11,4 +14,6 @@
     ) %>
     or <a href="#">join us in Slack</a>.
   </p>
-</section>
+</header>
+
+<img class="banner-image" src="<%= static_path(@conn, "/images/illustration.svg") %>">
```
Comment (brendastorer) [inline web/templates/home_page/about.html.eex]: Should this image have alt text?

File: web/templates/home_page/recent_dribbbles.html.eex [hunk 2]
```diff
@@ -1,29 +1,25 @@
...
+<ol
+  class="section-content dribbbles"
+  data-role="dribbble-list"
+  data-access-token="<%= System.get_env("DRIBBBLE_ACCESS_TOKEN") %>"
+  data-bucket-id="<%= System.get_env("DRIBBBLE_BUCKET_ID") %>"
+  data-count="6"
+  >
+  <li class="dribbble" data-role="dribbble-shot" style="display:none">
+    <a class="dribbble-link" href="#">
+      <img class="dribbble-img" src="">
```
Comment (brendastorer) [inline web/templates/home_page/recent_dribbbles.html.eex]: Should this image have alt text?

File: web/templates/home_page/upcoming_meetups.html.eex [hunk 3]
```diff
@@ -1,23 +1,21 @@
...
-          <time data-role="date-time">2/7/15, 6:30PM</time>
-          &bull;
-          <span data-role="members">97 Members Going</span>
-        </p>
-    </li>
-  </ol>
-</section>
+<ol class="section-content" data-role="meetup-list">
+  <li class="meetup-slat" data-role="meetup">
+      <h4 class="meetup-name" data-role="group-name">Meetup Group Name</h4>
```
Comment (brendastorer) [inline web/templates/home_page/upcoming_meetups.html.eex]: Is this indented one too far?

File: web/static/css/_grid-container.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  @include margin(null auto);
+  @include padding(null $small-spacing);
+  max-width: 64em;
+
+  @include media($medium-screen-up) {
+    @include padding(null $large-spacing);
+  }
+
+  @include media($grid-breakpoint) {
+    display: grid;
```
Comment (brendastorer) [inline web/static/css/_grid-container.scss:11]: Do have need for a fallback or are you comfortable using CSS Grid only? Do you know what the layout looks like without CSS Grid?

**Review by brendastorer — COMMENTED** [review]
A few small things but :tada: :tada:

File: web/static/css/_grid-container.scss (hunk 4 above)
Comment (creuter) [reply web/static/css/_grid-container.scss:11]: I'm comfortable with it, it's a website for designers. It's still readable without it, though, basically just a scaled up mobile view

**Comment by creuter** [comment]
Thanks!

**Pull request merged**